Formula/pcp.rb

```diff
@@ -0,0 +1,54 @@
+class Pcp < Formula
+  desc "Performance Co-Pilot is a system performance and analysis framework"
+  homepage "https://pcp.io/"
+  url "https://dl.bintray.com/pcp/source/pcp-4.1.3.src.tar.gz"
```

Comment:
Please use the github release tarballs, if possible, as this allows us to automatically check for updates

Reply:
Resolved this...